pkg/app/app.go

```diff
@@ -65,41 +65,40 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	logrus.Debug("Checking validation of provider %s", provider)
 
 	// OpenShift specific flags
-	deploymentConfig := cmd.Flags().Lookup("deployment-config").Changed
```

Comment:
yes, but opt will NOT check to see if the user provided anything _at all_. By checking if they passed a parameter in the first place, it'll fail out. For example. This will NOT fail if a user selects kubernetes and then passes in `--deployment-config=false` in the command line.